.github/workflows/main.yaml

```diff
@@ -171,7 +171,7 @@ jobs:
           push: true
           tags: |
             ${{ format('{0}/{1}:{2}', env.REGISTRY, env.IMAGE_NAME, needs.create-release.outputs.tag_name) }}
-            ${{ !startsWith(github.ref, 'refs/tags/nightly') && format('{0}/{1}:{2}', env.REGISTRY, env.IMAGE_NAME, 'latest') }}
+            ${{ !startsWith(github.ref, 'refs/tags/nightly') && format('{0}/{1}:{2}', env.REGISTRY, env.IMAGE_NAME, 'latest') || '' }}
```

Comment:
```suggestion
            ${{ ( !startsWith(github.ref, 'refs/tags/nightly') && format('{0}/{1}:{2}', env.REGISTRY, env.IMAGE_NAME, 'latest') ) || ''  }}
```

Reply:
lets try to be exact

Reply:
It should be exactly the same given `&&` operator has precedence over `||` operator (same with the `!` op.), but I will add the brackets for readability.